src/qubex/experiment/experiment.py

```
@@ -4108,12 +4145,14 @@
         self,
         target: str,
         *,
-        frequency_range: ArrayLike | None = None,
+        frequency_range: ArrayLike | None = np.arange(10.1, 10.7, 0.002),
```

Ruff

Do not perform function call `np.arange` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)